python/pyarrow/_parquet.pxd

```diff
@@ -135,6 +135,7 @@ cdef extern from "parquet/api/schema.h" namespace "parquet" nogil:
         int num_columns()
 
     cdef c_string FormatStatValue(ParquetType parquet_type, const char* val)
```

Comment:
How about removing the deprecated version?

Reply:
will do